sdk/identity/identity/src/credentials/interactiveBrowserCredential.ts

```diff
@@ -66,6 +65,12 @@ export class InteractiveBrowserCredential implements TokenCredential {
       this.redirectUri = "http://localhost";
     }
 
+    const url = new URL(this.redirectUri);
```

Comment:
I believe this won't work on older Nodes. You can see how I do this in `core-https`: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-https/src/util/url.ts